lua/ttt2/libraries/items.lua

```diff
@@ -172,7 +172,7 @@ function items.Get(name, retTbl)
 		local base = items.Get(retval.Base)
 
 		if not base then
-			Msg("ERROR: Trying to derive item " .. tostring(name) .. " from non existant item " .. tostring(retval.Base) .. "!\n")
+			ErrorNoHaltWithStack("ERROR: Trying to derive item " .. tostring(name) .. " from non existant item " .. tostring(retval.Base) .. "!\n")
```

Comment:
As this is also only internally called, I'm not sure a stack is any helpful here

Reply:
If it happens at all, it should be fixed, right? Some indication of how someone got there is important. I definitely utilize this inside PotOfGreedier.